test/cypress/integration/admin/advanced.spec.ts

```diff
@@ -0,0 +1,298 @@
+import { Chance } from 'chance'
+
+const CH = new Chance()
+const ALPHA_POOL = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+
+describe('Admin > Advanced Features', () => {
+  beforeEach(() => cy.login(Cypress.env('ADMIN_USERNAME'), Cypress.env('ADMIN_PASSWORD')))
+
+  describe('Given a WP user with permissions', () => {
+
+    describe('When the user activates the plugin and navigates to the settings page', () => {
+      beforeEach(() => 
+        cy.task('db:setup')
+          .activatePlugin('w3w-autosuggest')
+          .openSettingsPage('what3words')
+      )
+
+      it('Then the advanced features menu is collapsed', () => {
+        cy.getElementByDataTestId('advanced_form')
+          .should('exist')
+          .should('not.be.visible')
+      })
+
+      it('Then the advanced features menu should expand when toggle is clicked', () => {
+        cy.toggleAdvancedSettings()
+          .getElementByDataTestId('advanced_form')
+            .should('exist')
+            .should('be.visible')
+          .getElementByDataTestId('enable_placeholder')
+            .should('exist')
+            .should('be.visible')
+          .getElementByDataTestId('placeholder')
+            .should('exist')
+            .should('be.visible')
```

Comment:
Minor but checking for both existence and visibility is unnecessary. Just checking for visibility would be enough

Reply:
Nice 👍